docs/DeveloperGuide.md

```diff
@@ -55,13 +55,30 @@ and eventually update the UI which is displayed back to the user. This would con
 which would result in the latest data stored in DataStorage being saved into the plaintext files.
 
 ### UI Component
+**API** `Ui.java`
+Here's a class diagram of the `Ui` component
 ![](https://github.com/AY2223S2-CS2113-T13-4/tp/blob/master/docs/uml-diagrams/Ui.png?raw=true)
 
+How the `Ui` component works:
+1. When a command is entered by the user, `Parser` will create the relevant subclass of `Command`
+2. If the command is invalid, `Ui` will be called to print an error message.
+3. On the other hand, `Ui` will print a confirmation/success message after the execution of a task.
+
 ### Parser Component
 **API:** `Parser.java`
 Here's a class diagram of the `Parser` component
 ![](https://github.com/AY2223S2-CS2113-T13-4/tp/blob/master/docs/uml-diagrams/Parser.png?raw=true)
 
+Here's a sequence diagram of the `Parser` component
+![](https://github.com/AY2223S2-CS2113-T13-4/tp/blob/master/docs/uml-diagrams/Parser_Seq-Parser.png?raw=true)
```

Comment:
u wna put this after the how it works part to standardise w the Command Component part?
everything else looks fine!

Reply:
Okie!